### DISCUSSION TITLE
The taskbar gets cut off intermittently.

### DISCUSSION THREAD

**Discussion by shoooOny** [valinet/ExplorerPatcher#4019] [Q&A]
I want to report this because it seems like a bug, but I don't have space to do so, so I'm writing it here.

When I turn PC or monitor off and on

<img width="1919" alt="화면 캡처 2024-12-09 105627" src="https://github.com/user-attachments/assets/10b75e81-e153-4ea6-9f47-79c8d3edb8e2">

The Windows button size is reduced and the taskbar is also reduced, as if changed the taskbar icon size to 'small' in Settings. However, as you can see, the other icons are cut off, but their size remains the same.

<img width="1919" alt="화면 캡처 2024-12-09 105632" src="https://github.com/user-attachments/assets/9930bb20-87ff-4060-b7f5-2c8467a67e39">
If I take an action like unlocking the taskbar, it goes back to normal, but I can't do this every time.

using win11 24h2 (build 26100.2454) with latest ExplorerPatcher.

**Comment by Amrsatrio** [comment]
Yes I know, this is very annoying. I'd like not to be doing this every time as well. I tried finding out why but it seems like there's something weird outside the taskbar code that I couldn't figure out. This also happens on the ep_taskbar build for Windows 10. I may find a hacky fix for this which would make the DPI change handling deviate from the original implementation by Microsoft. Once that happens, it'll be mentioned in the release notes and I'll let you know.
